src/algorithms/math/horner-method/README.md

```diff
@@ -0,0 +1,16 @@
+# Horner's Method
+
+In mathematics, Horner's method (or Horner's scheme) is an algorithm for polynomial evaluation.
+With this method, it is possible to evaluate a polynomial with only n additions and n multiplications.
+Also, its storage requirements are only n times the number of bits of x.
```

Comment:
I think the description would highly benefit from a comparison between Horner's scheme to a regular by-definition approach; it would also give meaning to the repeated words “only” in both sentences.

Reply:
Thank you, Lazar. I've added your suggestions. I removed the repeated "only" words and tried to make the text more clear. Also, gave comparison scenarios for a specific polynomial showing that Horner's rule would use a smaller number of mathematical operations when compared to a simple by-definition approach.